Events/Ch1.event

```diff
@@ -92,7 +92,7 @@ UNIT Chiva Archer Vedge Level(1, Ally, 0) [10,1] 0x00 0x00 0x0 0x00 [0x00] NoAI
 UNIT
 
 CoinChick:
-UNIT Anna Thief Vedge Level(4, Ally, 0) [2,13] 0x00 0x00 0x0 0x00 [IronSword, Lockpick] NoAI
+UNIT Anna Thief Vedge Level(4, Ally, 0) [1,2] 0x00 0x00 0x0 0x00 [IronSword, Lockpick] NoAI
 UNIT
 
 Bad:
```

Comment:
Level 4 for the average enemy is a bit intense for Chapter 1. Could you change most of this to level 3, with maybe a few lv 4 enemies? Make Anna lv3 as well.